static/js/unread.js

```diff
@@ -264,9 +264,42 @@ exports.num_unread_for_person = function (user_ids_string) {
     return unread_privates.get(user_ids_string).num_items();
 };
 
+// Returns the next unread topic,
+// To select the selected topic pass select_this_topic as true
+exports.next_topic = function (selected_stream, selected_topic) {
+    // Get ordered stream list`
+    stream_list.build_stream_list();
+    var streams = stream_list.stream;
+
+    var topic_res = exports.unread_topic_counter.get_counts().subject_count;
+
+    // the topic list already contains only those which are unread
+    function has_unread_messages() {
+        return true;
+    }
+
+    function get_topics(stream) {
+        var topic_list = [];
+        var get_topic_list = topic_res.get(stream);
+        if (get_topic_list === undefined) {
+            return topic_list;
+        }
+        var topic_items = get_topic_list.items();
+        for (var topic in topic_items) {
+            if (topic_items[topic][1] > 0) {
+                topic_list.push({ stream : stream, topic : topic_items[topic][0]});
+            }
+        }
+        return topic_list;
+    }
+
+    return topic_generator.next_topic(streams, get_topics, has_unread_messages,
+      selected_stream, selected_topic);
```

Comment:
One argument per line, 4-spaces indented from the start of the `return`, is a pretty common pattern for something like this where the function has a really long name.  You can find plenty of examples if you just look around.